packages/generators/app/lib/utils/check-requirements.js

```diff
@@ -17,7 +17,7 @@ module.exports = function checkBeforeInstall() {
   }
 
   // warn if not using a LTS version
-  else if (semver.satisfies(currentNodeVersion, '14.x.x' || '15.x.x' || '17.x.x || 19.x.x')) {
+  else if (semver.satisfies(currentNodeVersion, '14.x.x || 15.x.x || 17.x.x || 19.x.x')) {
```

Comment:
just a though, does it make sense not just to do a parse of the currentNodeVersion to determine if `V.xx.xx` is odd...?

Reply:
Yeah, that would save updating this every time. I'll add that.